### PR TITLE
update-22.01-28

### PR DESCRIPTION
**Pull request by jayam04**

### CODE REVIEW

**Comment by jayam04** [comment]
we would merge it on 28th jan

**Comment by PowerMacFan** [comment]
ok. also i am back.

**Comment by jayam04** [comment]
have some work for you in main readme.md

**Comment by jayam04** [comment]
THANKS!

**Comment by PowerMacFan** [comment]
Your Welcome!

File: PyPrograms/contacts/contacts.py [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+
+# start of code
 # defining array
 fname = ["Jym"]
 lname = ["Patel"]
 number = [""]
 email = ["jympatel@yahoo.com"]
-    
+
 print("CLOSING PROGRAM WILL RESET ALL CHANGES MADE IN YOUR CONTACT LIST\n##")
```
Comment (PowerMacFan) [inline PyPrograms/contacts/contacts.py:15]: Perhaps we can use the python module pickle to store data? Its a bit hard to use but it would work.

**Review by PowerMacFan — APPROVED** [review]
Nice. Also I think that the new equations is really good.

File: PyPrograms/contacts/contacts.py (hunk 1 above)
Comment (jayam04) [reply PyPrograms/contacts/contacts.py:15]: I don’t know about pickle, I just know basics. I need time to work on Pickle

Comment (jayam04) [reply PyPrograms/contacts/contacts.py:15]: can you add pickle in this update, if you have time to do so.

Comment (PowerMacFan) [reply PyPrograms/contacts/contacts.py:15]: next update. I still got to learn pickle.

**Comment by PowerMacFan** [comment]
Its jan 28, can we merge now?

File: PyPrograms/contacts/contacts.py (hunk 1 above)
Comment (jayam04) [reply PyPrograms/contacts/contacts.py:15]: Ok. That’s fine, I also wanted pickle on next update

**Comment by jayam04** [comment]
Yes, we can merge now.

**Pull request merged**